app/views/admin/users/widgets/_credits.html.erb

```diff
@@ -30,7 +30,7 @@
                          size: 5,
                          min: 1,
                          max: 9999 %>
-      <%= f.label :credit_delta, "credits", class: "color-secondary flex-1", "aria-label": "Amount of credits to add or remove" %>
+      <%= f.label :credit_amount, "credits", class: "color-secondary flex-1", "aria-label": "Amount of credits to add or remove" %>
```

Comment:
This got missed during a refactor in #16174. This fixes the label being linked to the textbox.